test/Vest.t.sol

```diff
@@ -53,8 +60,9 @@ contract VestTest is BaseTest {
         vm.warp(TWENTY_YEARS + OFFSET);
     }
 
-    function testOwner() public {
-        assertEq(vest.owner(), address(this));
+    function testOwner(address owner, address) public {
```

Comment:
not sure why there is a second `address` parameter to `testOwner`

Reply:
Mistake on my side, we'll correct it!